docs/source/traitsui_user_manual/testing/tutorials/test_with_nested_object.rst

```diff
@@ -21,7 +21,7 @@ Suppose we have a GUI application like this::
         pay_check = Instance(PayCheck)
 
         def _pay_check_default(self):
-            return PayCheck(pay=self.min_hourly_wage, n_hours=self.n_hours)
```

Comment:
Thank you. As I was writing this, I wanted to bring in doctest, for exactly something like this, but thought the setup would be too much to review. I ran all the examples, and found this exact bug, but what I neglected to do was to copy the fix back from my local script to the documentation.